contract-ui/tabs/overview/page.tsx

```diff
@@ -1,40 +1,256 @@
-import { Flex } from "@chakra-ui/react";
+import { BuildYourApp } from "./components/BuildYourApp";
+import { ShareContract } from "./components/ShareContract";
+import { Divider, Flex, GridItem, Icon, SimpleGrid } from "@chakra-ui/react";
+import { HiOutlineBookOpen } from "@react-icons/all-files/hi/HiOutlineBookOpen";
 import { useContract } from "@thirdweb-dev/react";
 import { Abi } from "@thirdweb-dev/sdk";
+import { ContractType } from "@thirdweb-dev/sdk/evm";
 import { useContractFunctions } from "components/contract-components/hooks";
 import { ImportContract } from "components/contract-components/import-contract";
 import { ContractFunctionsOverview } from "components/contract-functions/contract-functions";
-import { Heading } from "tw-components";
+import { Heading, Link, Text } from "tw-components";
 
 interface CustomContractOverviewPageProps {
   contractAddress?: string;
+  contractType?: ContractType;
 }
 
+const GUIDES = {
```

Comment:
we're doing this by contract type when I believe we should do it by detected extensions, so it's more scalable and works for custom contracts too

Reply:
contract type is what we'll start with, we'll move it over to extension later needs a lot more thinking on what extension = what guide (guides are typically based on our contracts), so I would consider this fine for now